src/BackgroundLocationTracking/Platforms/Android/NotificationHelper.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Android.App;
+using Android.Content;
+using AndroidX.Core.App;
+
+namespace BackgroundLocationTracking
+{
+    public class NotificationHelper
```

Comment:
Let's keep notifications out of the sample. We discussed instead having the map upgrade a graphics overlay with the location history

Reply:
As we discussed, we might need this persistent notification to keep the status of foreground service on notification bar.